migrator/migrator-servlet/src/main/webapp/index.jsp

```diff
@@ -1,23 +1,94 @@
 <!DOCTYPE html>
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
-<%@ page import="com.google.phonenumbers.HelloWorld" %>
+<%@ page import="com.google.phonenumbers.ServletMain" %>
+<%@ page import="com.google.phonenumbers.migrator.MigrationJob" %>
+<%@ page import="java.io.IOException" %>
+<%
+  final String E164_NUMBERS_LINK = "https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers";
+  final String COUNTRY_CODE_LINK = "https://countrycode.org/";
+  final String DOCUMENTATION_LINK = "./"; // TODO: use README documentation link when uploaded
+%>
 <html>
 <head>
-  <link type="text/css" rel="stylesheet" href="/stylesheets/hello-world.css" />
-  <title>Hello World</title>
+  <link type="text/css" rel="stylesheet" href="/stylesheets/servlet-main.css" />
+  <title>Migrator</title>
 </head>
 <body>
-    <h1>Hello World</h1>
+  <div class="page-heading">
+    <h1>Phone Number Migrator</h1>
+    <p>
+      Enter a path to a text file containing <a href="<%=E164_NUMBERS_LINK%>" target="_blank">E.164 phone numbers</a>
+      from the same country or a single E.164 number as well as the corresponding BCP-47 <a href="<%=COUNTRY_CODE_LINK%>" target="_blank">country code</a>
+      to potentially migrate stale phone numbers to valid formats based on the given country code. For more information
+      on the capabilities of the migrator as well as instructions on how to install the command line tool, please
+      visit view the <a href="<%=DOCUMENTATION_LINK%>" target="_blank">documentation</a>.
+    </p>
+  </div>
 
-  <p>This is <%= HelloWorld.getInfo() %>.</p>
-  <table>
-    <tr>
-      <td colspan="2" style="font-weight:bold;">Available Servlets:</td>
-    </tr>
-    <tr>
-      <td><a href='/hello'>Hello App Engine</a></td>
-    </tr>
-  </table>
+  <div class="migration-result">
+    <%
+      if (request.getAttribute("numberError") == null && request.getAttribute("number") != null) {
+        if (request.getAttribute("validMigration") != null) {
+          out.print("<h3 class='valid'>Valid +" +request.getAttribute("numberCountryCode") + " Phone Number Produced!</h3>");
+          out.print("<p>The stale number '" + request.getAttribute("number") + "' was successfully migrated into the" +
+                  " phone number: +" + request.getAttribute("validMigration") + "</p>");
+        } else if (request.getAttribute("invalidMigration") != null) {
+          out.print("<h3 class='invalid-migration'>Invalid +" +request.getAttribute("numberCountryCode") + " Migration</h3>");
```

Comment:
Maybe add 'for country code: request.getAttribute("numberCountryCode")' or something similar to make it more clear. 

Here and the rest of the cases.